sarracen/sarracen_dataframe.py

```diff
@@ -17,6 +18,50 @@ def __init__(self, data=None, params=None, *args, **kwargs):
         self._units = None
         self.units = Series([np.nan for i in range(len(self.columns))])
 
+        # First look for 'x', then 'rx', and then fallback to the first column.
+        if 'x' in data.columns:
+            self._xcol = 'x'
+        elif 'rx' in data.columns:
+            self._xcol = 'rx'
+        else:
+            self._xcol = data.columns[0]
+
+        # First look for 'y', then 'ry', and then fallback to the second column.
+        if 'y' in data.columns:
+            self._ycol = 'y'
+        elif 'ry' in data.columns:
+            self._ycol = 'ry'
+        else:
+            self._ycol = data.columns[1]
```

Comment:
Let's break this into its own private method to keep the constructor slim. 

_identify_position_columns() maybe? _find_spatial_columns()?